specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/examples/2018-01-01/call_participant_removed.json

```diff
@@ -0,0 +1,45 @@
+{
+  "source": "/subscriptions/{subscription-id}/resourceGroups/{group-name}/providers/Microsoft.Communication/communicationServices/{communication-services-resource-name}",
+  "subject": "thread/{thread-id}/sender/{rawId}/recipient/{rawId}",
+  "type": "Microsoft.Communication.CallParticipantRemoved",
+  "time": "2021-02-19T00:25:59.9436666Z",
+  "id": "02272459-badb-4e2e-b538-4cb8a2f71da6",
+  "data": {
+    "reason": {
+      "code": 1,
+      "subCode": 1,
+      "phrase": "Reason Code"
+    },
+    "user": {
+      "communicationIdentifier": {
+        "kind": "communicationUser",
+        "rawId": "8:acs:d9ff4983-39e7-4420-8d54-f6354ed06ccd_7a668e62-08f4-4ac1-b29e-f01010f57475",
+        "communicationUser": {
```

Comment:
update the properties "user" and "startedBy" to reflect the callparticipant model for the calling event examples
![image](https://github.com/user-attachments/assets/7e0fc314-178a-4985-bc36-7a44e67366f6)